packages/lodestar/src/network/gossip/gossipsub.ts

```diff
@@ -431,6 +431,10 @@ export class Eth2Gossipsub extends Gossipsub {
         }
       }
     }
+
+    // track gossip peer score
+    const gossipScore = Array.from(this.peers.keys()).map((peer) => this.score.score(peer));
+    metrics.gossipPeer.score.set(gossipScore);
```

Comment:
I think you can achieve the same results with regular Gauges, without a new metric type

```ts
    let peerCountScoreGraylist = 0;
    let peerCountScorePublish = 0;
    let peerCountScoreGossip = 0;
    let peerCountScoreMesh = 0;
    for (const peerIdStr of this.peers.keys()) {
      const score = this.score.score(peerIdStr);
      if (score > gossipScoreThresholds.graylistThreshold) peerCountScoreGraylist++;
      if (score > gossipScoreThresholds.publishThreshold) peerCountScorePublish++;
      if (score > gossipScoreThresholds.gossipThreshold) peerCountScoreGossip++;
      if (score > 0) peerCountScoreMesh++;
    }
    metrics.gossipPeer.scoreByThreshold.set({threshold: "graylist"}, peerCountScoreGraylist);
    metrics.gossipPeer.scoreByThreshold.set({threshold: "publish"}, peerCountScorePublish);
    metrics.gossipPeer.scoreByThreshold.set({threshold: "gossip"}, peerCountScoreGossip);
    metrics.gossipPeer.scoreByThreshold.set({threshold: "mesh"}, peerCountScoreMesh);
```

Reply:
We should also track the AvgMinMax of all scores, to answer to different questions:
- `scoreByThreshold` label by threshold: How many peers are in each gossip score threshold?
- `score` AvgMinMax: How well or bad scored are the peers numerically?